src/ucs/core/models.py

```diff
@@ -69,6 +69,10 @@ class UploadAttempt(BaseModel):
         ..., description="The public key used by the submittter to encrypt the file."
     )
     model_config = ConfigDict(from_attributes=True, title="Multi-Part Upload Details")
+    s3_endpoint_alias: str = Field(
```

Comment:
Just by convention the core should not name the technology:
```suggestion
    storage_alias: str = Field(
```

Reply:
Probably needs some changes in the across other services as well.
Now only the outbound event payload knows that it's relating to S3 and it's storage_alias everywhere else